app/screens/home/recent_mentions/components/empty.tsx

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import React from 'react';
+import {View} from 'react-native';
+
+import {useTheme} from '@app/context/theme';
+import {changeOpacity, makeStyleSheetFromTheme} from '@app/utils/theme';
+import {typography} from '@app/utils/typography';
+import FormattedText from '@components/formatted_text';
+
+// @ts-expect-error svg extension
+import Mention from './mention_icon.svg';
```

Comment:
this is fine, but have you confirmed with UX that this illustration does not changes some of its colors based on the theme?

cc: @matthewbirtch @Rina-dsg